doc/source/dev/roadmap-detailed.rst

```diff
@@ -444,10 +434,11 @@ The following improvements will help SciPy better serve this role.
 
 In addition, we should:
 
+- Consistently handle ``nan_policy`` and ``axis`` arguments in all ``stats``
+  functions (where appropriate).
 - Continue work on making the function signatures of ``stats`` and
   ``stats.mstats`` more consistent, and add tests to ensure that that
   remains the case.
-- Improve statistical tests: consistently provide options for one- and
-  two-sided alternative hypotheses where applicable, return confidence
-  intervals for the test statistic, and implement exact p-value calculations -
-  considering the possibility of ties - where computationally feasible.
+- Improve statistical tests: return confidence intervals for the test
+  statistic, and implement exact p-value calculations - considering the
+  possibility of ties - where computationally feasible.
```

Comment:
I think the `alternative` parameter is implemented consistently enough to remove this from the roadmap. See gh-12506.

Reply:
agreed